pkg/controller/perconaservermongodb/psmdb_controller.go

```diff
@@ -797,7 +798,7 @@ func (r *ReconcilePerconaServerMongoDB) ensureSecurityKey(ctx context.Context, c
 func (r *ReconcilePerconaServerMongoDB) deleteOrphanPVCs(ctx context.Context, cr *api.PerconaServerMongoDB) error {
 	for _, f := range cr.GetFinalizers() {
 		switch f {
-		case "delete-psmdb-pvc":
+		case naming.FinalizerDeletePVC, "delete-psmdb-pvc":
```

Comment:
should we print deprecation log for `delete-psmdb-pvc`? 

P.S.: you should be able to do something like this rather than duplicating the code:

```suggestion
		case  "delete-psmdb-pvc":
                     log.Info("deprecation message")
                     fallthrough
		case naming.FinalizerDeletePVC:
```

Reply:
So I can fix it like this, that it is not a problem. I don't know how it's better to do. My logic was - make it as easier as possible for refactoring in the future. So when we need to delete "delete-psmdb-pvc", we just can delete it without any problems. 
@egegunes